package.json

```diff
@@ -19,7 +19,7 @@
     "http-status-codes": "^2.1.4",
     "launchdarkly-js-client-sdk": "^2.20.0",
     "nuxt": "^2.15.3",
-    "sbc-common-components": "^2.4.34"
+    "sbc-common-components": "^2.4.37"
```

Comment:
This version includes the necessary SbcHeader etc changes (to sync the account ID when the target app loads).